src/manager/frontend/src/components/sections/sidebar.tsx

```diff
@@ -7,20 +7,20 @@ export function SideBar() {
         <div className="hidden bg-muted/40 md:block pt-4">
             <div className="flex-1">
                 <nav className="grid items-start px-2 text-sm font-medium lg:px-4">
-                    <Link
+                    <a
```

Comment:
any idea why we can't use Link?

Reply:
I just feel that the Link is nextjs is a bit complicated, i ran into Link preload issue with exported mode so i  just switched to use <a> instead. (https://github.com/vercel/next.js/discussions/49607)